.gitignore

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2010 - 2022 Daniel Stenberg, <daniel@haxx.se>, et al.
```

Comment:
IANAL, but I don't believe this for of copyright notice qualifies as a UCC notice, which might affect copyright enforcement in some countries.

Reply:
Do we really *need* to use `SPDX-FileCopyrightText:` ? It'll create a terribly annoying duplicated set of information (as I too believe we should keep the original "full" copyright line).

Reply:
Ah, I didn't cover that in full detail in the second commit. `SPDX-FileCopyrightText` is somewhat recommended because it is a unique, ISO standardised identifier for communicating copyright, and does not trigger false-positives in tools. However, as covered in [this FAQ item](https://reuse.software/faq/#standard-copyright) you are free to stick with the "traditional" copyright lines. In my subsequent commits I did not convert it.